core/src/main/java/com/klaytn/caver/methods/response/GovernanceChainConfig.java

```diff
@@ -267,6 +273,12 @@ public static class Reward {
             @JsonProperty("ratio")
             private String ratio;
 
+            /**
+             * The value of kip82ratio field.
+             */
+            @JsonProperty("kip82ratio")
+            private String kip82ratio;
```

Comment:
How about change kip82ratio to kip82Ratio because of using in getter and setter method same as came case pattern. (getKip82Ratio, setKip82Ratio)

Reply:
Right, It is typo. Resolved it in 949fcff